include/sys/types.h

```diff
@@ -29,7 +29,11 @@ typedef double double_t;
 
 #ifndef _HALF_FLOAT_T
 #define _HALF_FLOAT_T
-typedef short half_t;           /* IEEE16 half float type */
+#ifdef __SDCC
```

Comment:
I think this is probably best as `#ifndef SCCZ80` - I'm playing around with VScode and using clang as the IntelliSense engine.

Reply:
Done. Added `f32_t` and `f16_t`, to match the integer shortcuts. Not sure what to do about 48 and 64 bit math, though. Another day.

Reply:
Took the `f32_t` and `f16_t` shortcuts back out. `f32_t` can't work consistently.